alma/alma-installments-prestashop
# [cart-item] Bug shield icon does not match design

### Reason for change



[Linear task](https://linear.app/almapay/issue/ECOM-1684/[🪲-bug]-shield-icon-does-not-match-design)

### Code changes


We remplace the shield icon from design

### How to test

_As a reviewer, you are encouraged to test the PR locally._


Active insurance in our module
Add product with insurance
Add same product without insurance
Check in the cart page if there is the right shield icon

### Checklist for authors and reviewers




### Non applicable
